README.md

```diff
@@ -11,6 +11,129 @@ For **questions** or **general comments** regarding the use of this repository,
 
 And you can check all of our open-source projects at [inaka.github.io](http://inaka.github.io)
 
+
+
+## Conventions & Rules
+
+#### Use snake_case with JSON
+> When the endpoint returns a JSON, the keys should always be in snake_case.
+
+
+##### Examples
+###### Good
+
+      {
+      "caption": "string",
+      "comment_count": 0,
+      "created_at": "2017-03-07T18:57:44.622Z",
+      "id": "string",
+      "like_count": 0,
+      "media_type": "image",
+      "owner": "string"
+    }
+
+###### Bad
+
+      {
+      "caption": "string",
+      "commentCount": 0,
+      "createdAt": "2017-03-07T18:57:44.622Z",
+      "id": "string",
+      "likeCount": 0,
+      "mediaType": "image",
+      "owner": "string"
+    }
+
+##### Reasoning
+Even though in some programming languages is common to name variables or object instances with camelCase names, we consider the snake_case is the most readable and understandable way to name the key fields on a JSON object
+
+---
+
+#### Don't use response envelopes by default
+> When the endpoint returns, for example, an array, it should be returned as a plain array, not a dictionary with the array as a value.
+
+
+##### Examples
+###### Good
+
+    [
+      {
+      "like_count": 0,
+      "id": "string",
+      "media_type": "comment",
+      "owner": "string"
+      },
+      {
+        "like_count": 0,
+        "id": "string",
+        "media_type": "comment",
+        "owner": "string"
+      },
+      {
+        "like_count": 0,
+        "id": "string",
+        "media_type": "comment",
+        "owner": "string"
+      }
+    ]
+
+###### Bad
+
+    { "items" : [
+        {
+          "like_count": 0,
+          "id": "string",
+          "media_type": "comment",
+          "owner": "string"
+        },
+        {
+          "like_count": 0,
+          "id": "string",
+          "media_type": "comment",
+          "owner": "string"
+        },
+        {
+          "like_count": 0,
+          "id": "string",
+          "media_type": "comment",
+          "owner": "string"
+        }
+      ]
+    }
+
+##### Reasoning
+If you're calling a **GET** on `/entities` , you're asking for entities, not an object with the entities inside one of its properties. Think about it as a christmas gift. If you ask santa for all the power rangers action figures ( and i'm talking about the 90's ones ) and you only see one box under the tree, you can't know if you're getting just one or all of them wrapped in one gift. _Also,  ALL the power rangers? Couldn't you just ask for ONE toy?_
+
+---
+
+#### Use all HTTP verbs ( Not everything is a GET )
```

Comment:
Are these spaces after and before the parenthesis on purpose? I would suggest removing them, and writing them the normal way: `Use all HTTP verbs (Not everything is a GET)`
Thoughts @elbrujohalcon ?